res/controllers/Hercules_DJControl_Jogvision-scripts.js

```diff
@@ -142,13 +142,15 @@ DJCJV.init = function(id) {
 
 // Finalization
 DJCJV.shutdown = function() {
+	print("Hercules DJControl Jogvision id: \""+id+"\" finalizing");
 	//Set all LED states to off
 	midi.sendShortMsg(DJCJV.Channel["[Channel1]"].deck, 0x7F, off);
 	midi.sendShortMsg(DJCJV.Channel["[Channel2]"].deck, 0x7F, off);
 	if (ledRotationTimer) {
 		engine.stopTimer(ledRotationTimer);
 		ledRotationTimer = 0;
 	}
+	print("Hercules DJControl Jogvision id: \""+id+"\" finalized");
```

Comment:
`id` should be undefined here, since it's only passed to `init()`, not to `shutdown()`. Please revert this commit.

Reply:
ping